### PR TITLE
Use octokit action rather than requests

### PR DESCRIPTION
**Pull request by jonmoss-konsentus**
Corrected plurals for step -> steps

### CODE REVIEW

File: dist/index.js [hunk 1]
```diff
@@ -26934,7 +26958,7 @@ module.exports = require("util");
 /***/ (function(module, __unusedexports, __webpack_require__) {
 
 const { convertToUtc, dateDiff } = __webpack_require__(483);
-const { parseStep } = __webpack_require__(743);
+const { parseSteps } = __webpack_require__(461);
```
Comment (andymed-jlp) [inline dist/index.js:26961]: This seems a really strange magic number - I presume it's some hash or something and is generate for us? I've no problem with it - just seems strange :-)

File: src/parseSteps/index.js [hunk 2]
```diff
@@ -1,8 +1,8 @@
 const { convertToUtc, dateDiff } = require('../helpers');
 
-const parseStep = unparsedStep => {
-  const { name, status, number, started_at, completed_at, conclusion } = unparsedStep;
-  const parsedStep = {
+const parseSteps = unparsedSteps => {
```
Comment (andymed-jlp) [inline src/parseSteps/index.js:3]: Nice. I like consistency of plurals! :-)

**Review by andymed-jlp — APPROVED** [review]
I don't know the intricacies of octokit, but this all seems good to me.

File: dist/index.js (hunk 1 above)
Comment (jonmoss-konsentus) [reply dist/index.js:26961]: I haven't really questioned how ncc creates this file and why it does pick these seemingly arbitrary numbers, it is very stage!

**Pull request merged**